.github/workflows/webpack.yml

```
@@ -0,0 +1,29 @@
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)